tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -242,6 +242,8 @@ function tag_pr($payload, $opened) {
 	if($opened) {	//you only have one shot on these ones so as to not annoy maintainers
 		$tags = checkchangelog($payload);
 
+		if(strpos(strtolower($title), 'log') !== FALSE || strpos(strtolower($title), 'logs') !== FALSE || strpos(strtolower($title), 'logging') !== FALSE)
```

Comment:
`logs` and `logging` are included by `log`

Reply:
will it include things like "catalog" and "logic"? If so that's kind of annoying

Reply:
Yeah